templates/layout.html

```diff
@@ -110,25 +110,97 @@
           {%- endif -%}
         {%- endif -%}
         </li>
-     <li class="nav-item dropdown dropdown-pull-right">
-        <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
+      <!--This css added for fix a bug for language menu modal creation
+            By: Gagan Aggarwal
+            Email: gaggarwal124@gmail.com -->
+      <style>
+        body {
+          font-family: Arial, Helvetica, sans-serif;
+        }
+
+        .modal {
+          display: none;
+          position: fixed;
+          z-index: 1;
+          padding-top: 100px;
+          left: 0;
+          top: 0;
+          width: 100%;
+          height: 100%;
+          overflow: auto;
+          background-color: rgb(0, 0, 0);
+          background-color: rgba(0, 0, 0, 0.4);
+        }
+
+        /* Modal Content */
+        .modal-content {
+          background-color: #fefefe;
+          margin: auto;
+          padding: 20px;
+          border: 1px solid #888;
+          width: 80%;
+        }
+
+        /* The Close Button */
+        .close {
+          color: #aaaaaa;
+          float: right;
+          font-size: 28px;
+          font-weight: bold;
+        }
+
+        .close:hover,
+        .close:focus {
+          color: #000;
+          text-decoration: none;
+          cursor: pointer;
+        }
+      </style>
+
+      <li class="nav-item dropdown dropdown-pull-right">
+        <a id="myBtn" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button"
+          aria-haspopup="true" aria-expanded="false">
           <i class="fa fa-globe" aria-hidden="true"></i> <span class="caret"></span>
         </a>
-        <div class="dropdown-menu" aria-labelledby="dropdown01">
-          {%- set eng_active = this.alt == 'en' %}
-          <a class="dropdown-item {% if eng_active %}active{% endif %}" href="{{ '.'|url(alt='en') }}">
-          English
-          </a>
-
-          {%- for alt, alternative in config.ALTERNATIVES.items()|sort -%}
-          {%- set alt_active = alt == this.alt -%}
-          {%- if alt != config.primary_alternative -%}
-          <a class="dropdown-item {% if alt_active %}active{% endif %}" href="{{ '.'|url(alt=alt) }}">
-          {{ transbag('translate', alt, alternative.name.en.lower()) }}
-          </a>
-          {%- endif -%}
-          {%- endfor -%}
+        <!-- Creating Modal Window Div -->
+        <div id="myModal" class="modal">
```

Comment:
"myModal" and "myBtn" aren't especially helpful ID names. If you're adding IDs, to elements, they should clearly identify their place on the page.